pkg/apis/componentconfig/types.go

```diff
@@ -137,6 +137,10 @@ type KubeletConfiguration struct {
 	// manifestURLHeader is the HTTP header to use when accessing the manifest
 	// URL, with the key separated from the value with a ':', as in 'key:value'
 	ManifestURLHeader string `json:"manifestURLHeader"`
+	// manifestURLHeaders is a comma-separated list of HTTP headers to use
+	// when accessing the manifest URL, with the key separated from the value
+	// with a ':', as in 'key:value', will be combined with ManifestURLHeader
+	ManifestURLHeaders []string `json:"manifestURLHeaders"`
```

Comment:
nit: /s/`, will be combined with ManifestURLHeader`/`. These will be applied in addition to the header provided via ManifestURLHeader.`